CodeSniffer/Standards/Squiz/Tests/Commenting/DocCommentAlignmentUnitTest.php

```diff
@@ -38,25 +38,47 @@ class Squiz_Tests_Commenting_DocCommentAlignmentUnitTest extends AbstractSniffUn
      * The key of the array should represent the line number and the value
      * should represent the number of errors that should occur on that line.
      *
+     * @param string $testFile The name of the file being tested.
+     *
      * @return array<int, int>
      */
-    public function getErrorList()
+    public function getErrorList($testFile='')
```

Comment:
because there were already 2 test files. One testing PHP, one testing JS. Interestingly they were constructed in a way that they have the errors exactly on the same lines ... 

But we only want to add a test to the PHP file here, which makes the 2 tests diverge and we need to introduce the switch.